jkube-kit/jkube-kit-wildfly-jar/src/main/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricher.java

```diff
@@ -134,39 +154,16 @@ private boolean isAvailable() {
         return config.containsKey("cloud");
     }
 
-    protected int getFailureThreshold() {
-        return Configs.asInteger(getConfig(Config.FAILURETHRESHOLD));
-    }
-
-    protected boolean isProbeEnforced() {
-        return Configs.asBoolean(getConfig(Config.ENFORCEPROBES));
-    }
-
-    protected int getSuccessThreshold() {
-        return Configs.asInteger(getConfig(Config.SUCCESSTHRESHOLD));
-    }
-
-    protected String getScheme() {
-        return Configs.asString(getConfig(Config.SCHEME));
-    }
-
     protected int getPort() {
-        return Configs.asInt(getConfig(Config.PORT));
+        return asInt(getConfig(Config.PORT));
```

Comment:
Is this method still used?? I see we replaced the call to `getPort()` in the `discoverWildflyJARHealthCheck`, this method is probably useless now.

Reply:
This and the other two methods `getLivenessPath()` and `getReadinessPath()` are only required for this test
https://github.com/eclipse/jkube/blob/e643a0a28ffe520b3122a42cf01cf0792d4fd1ba/jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricherTest.java#L195-L201

Reply:
Do we really need this test? If yes then tell me how to test the actual Enricher behavior after removing these three methods.

Reply:
Testing an unused method seems like something unnecessary.
We should refactor the test to actually verify something. IF we changed the implementation of the calling method to evaluate the ports, then refactor the test to verify that method.

> Do we really need this test? 

That's part of your assignment. You should try to evaluate what the class is actually doing and if the tests are verifying this behavior.